Tiltfile

```diff
@@ -19,6 +19,7 @@ update_settings(max_parallel_updates = 10)
 # Runtime configuration
 config.define_bool("ci", False, "We are running in CI")
 config.define_bool("manual", False, "Set TRIGGER_MODE_MANUAL by default")
+config.define_bool("m1", False, "Build Solana from source for m1 arm64")
```

Comment:
Naming nit: how about "arm64"?  The build steps will work fine on non-M1 arm64 machines.